src/main/java/org/sagebionetworks/bridge/validators/AssessmentValidator.java

```diff
@@ -2,11 +2,7 @@
 
 import static org.apache.commons.lang3.StringUtils.isBlank;
 import static org.sagebionetworks.bridge.BridgeConstants.SHARED_APP_ID;
-import static org.sagebionetworks.bridge.validators.Validate.BRIDGE_EVENT_ID_ERROR;
-import static org.sagebionetworks.bridge.validators.Validate.BRIDGE_EVENT_ID_PATTERN;
-import static org.sagebionetworks.bridge.validators.Validate.CANNOT_BE_BLANK;
-import static org.sagebionetworks.bridge.validators.Validate.CANNOT_BE_NEGATIVE;
-import static org.sagebionetworks.bridge.validators.Validate.INVALID_TYPE;
+import static org.sagebionetworks.bridge.validators.Validate.*;
```

Comment:
The code convention for Bridge Server is that we don't use wild cards for imports. My IntelliJ config is import with * and static import with * is set to 999 (basically never). The sort order is

* import static
* (blank line)
* import java.*
* import javax.*
* (blank line)
* all other imports
* (blank line)
* import org.sagebionetworks.bridge.*

I can send you my code style XML if that's easier.